irods/test/connection_test.py

```diff
@@ -236,6 +239,18 @@ def test_assigning_session_connection_timeout__issue_569(self):
             sess, old_timeout
         )
 
+    def test_legacy_auth_codepath__issue_499(self):
```

Comment:
Let's specify that we're specifically testing the `legacy_auth.force_legacy_auth` configuration setting here. Maybe this name conveys that a little better?
```
test_legacy_auth_used_with_force_legacy_auth_configuration__issue_499
```

Reply:
Done